### DISCUSSION TITLE
Dropdown in Navigation Menu

### DISCUSSION THREAD

**Discussion by markwsac** [mearashadowfax/ScrewFast#249] [Q&A] [answered]
Thanks for this beautiful looking template. Is there any quick way to shop dropdown in navigation menu? Like products > product1, product2, product3.. Thanks in advance!

**Comment by mearashadowfax** [comment, marked as answer]
Hey @markwsac, thanks for the kind words!

Absolutely, you have a couple of options:

- You can use the `NavbarMegaMenu` component and adjust the `MegaMenuLink` to add a dropdown on hover (I've attached a screenshot for reference).

- Or you could implement your own dropdown using Preline UI’s Dropdown plugin. You can check it out here: [Preline Dropdown Plugin](https://preline.co/plugins/html/dropdown.html). 

Just add the following script to your setup:

```html
<script>
  import "@preline/dropdown/index.js";
</script>
```

![ScreenRecording2024-11-01220533-ezgif com-video-to-gif-converter](https://github.com/user-attachments/assets/357dc3e7-746b-4453-b209-d23d11b7e16b)

**Comment by markwsac** [comment]
Excellent. Thanks for your prompt response. Let me try. Would be great if you include it in the demo site as people compare different themes. Some beginners (like me) wonder if the theme has this functionality.. Your theme stands out in terms of looks. It deserves more visibility and love. Cheers!
